Add a tutorial for MetaShRIMPS (#4272)
galaxyproject/training-material: topics/ecology/tutorials/Metashrimps_tutorial/tutorial.md

---
layout: tutorial_hands_on

title: Creating FAIR Quality assessment reports and draft of Data Papers from EML
  metadata with MetaShRIMPS
zenodo_link: https://doi.org/10.5281/zenodo.8130567
questions:
- How to improve the FAIR quality of an EML metadata ?
- How to use metadata for machine actionnable processes ?
- What is the point of having a FAIR metadata ?
objectives:
- Learn how to use the interactive tool Metashrimps
- Understand the challenges MetaShRIMPS is trying to respond to
- Learn how to create a FAIR Quality assessment report from a metadata using EML standard
- Understand the concept of Data Paper and learn how to produce it
- Explain the necessity of using such tools when producing ecological metadata
time_estimation: 30mn
key_points:
- This tool aims to improve FAIR quality of metadata
- Creating metadata as FAIR as possible is a must
- Be carefull of the format and standard of metadata used only EML metadata will work
tags:
  - Metadata
  - EML
  - FAIR
  - Data Paper
contributors:
- TanguyGen

> - yvanlebras

Reviewer: If you like can use our fancy new contributions syntax to more accurately record contributions to a training material, declaring which people should be recorded for authorship or just editing.

```
contributions:
  authorship:
    - shiltemann
  editing:
    - bebatut
    - hexylena
  funding:
    - carpentries
  testing:
    - userX
  ux:
    - userY
  infrastructure:
    - userZ
```

see the [full gtn tutorial schema documentation](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/schemas/tutorial.html) for details on the individual roles.

@TanguyGen please also add your information to the CONTRIBUTORS.yaml so the tests can get a bit further :)

data/metadata to the production of a metadata with high degree of FAIRness.
The other objective of this tool is to highlight the work of all of the people that helped in producing this data by giving the access of a draft of Data Paper
that is reuseable either in a non editable HTML file, that can represent the metadata in a more ergonomic way facilitating its understanding and shareability,
or in an editable docx file. Having an editable Data Paper draft will allow the producer to complete/modify the draft of Data Paper so that it could become
publishable as a real Data Paper giving recognition to all the people that helped producing the data.

> <comment-title>What is a Data Paper?</comment-title>
> According to the [GBIF](https://www.gbif.org/data-papers) (Global Biodiversity Information Facility), 
>A data paper is a peer reviewed document describing a dataset, published in a peer reviewed journal. It takes effort to prepare, curate and describe data. 
>Data papers provide recognition for this effort by means of a scholarly article.
{:  .comment}
# Get data

> <hands-on-title> Data Upload </hands-on-title>
>
> 1. Create a new history for this tutorial

> > 2.Import this metadata file from [Zenodo]({{ https://doi.org/10.5281/zenodo.8130567 }}) to test it

Reviewer: ```suggestion
> 2. Import this metadata file from [Zenodo]({{ page.zenodo_link }}) to test it
```

> >     -> `{{ Training Data for "Creating Quality FAIR assessment reports and draft of Data Papers from EML metadata with MetaShRIMPS" }}`):

Reviewer: ```suggestion
>     -> Training Data for "Creating Quality FAIR assessment reports and draft of Data Papers from EML metadata with MetaShRIMPS"):
```

`{{ .. }}` marks a variable in Jekyll, here it will look for a variable with that name and not be able to find it. Please avoid `{{` outside of e.g. page.zenodo_link where you want to access a variable.

>    ```
>    https://zenodo.org/record/8130567/files/Kakila_database_marine_mammal.xml
>    ```
>
>    {% snippet topics/ecology/tutorials/Metashrimps_tutorial/import_files.md %}

# Upload Data in MetaShRIMPS

When oppening MetaShRIMPS, you will have an interface looking like this :

> <img src="./Images/upload_1.png" alt="FAIR Data Principles" width="500"/>

Reviewer: ```suggestion
![FAIR Data Principles](./Images/upload_1.png){:width="500"}
```

you can write it like this for markdown

Reviewer: It looks like this alt-text doesn't describe the image very well for users who are relying on a screen reader to access the content. We have a short guide on [what's appropriate for alt-text vs captions in the GTN Contributing Guide](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/create-new-tutorial-content/tutorial.html#guidelines-on-alt-vs-figcaption-text) which has more details. *This comment may apply to more images, please consider checking all of them.*

you want to use. 
> <warning-title>Select the right format</warning-title>
> The file uploaded in this tool must be a metadata in XML format using EML metadata standard.
{: .warning}

<img src="./Images/upload_2.png" alt="FAIR Data Principles" width="500"/>

After uploading the file, or if you have indicate it as input data if the tool, you just have to click on **Execute** to launch the tool with the file.

# Outputs

After clicking the **Execute** button, 2 new tabs called "Draft of Data Paper" and "Fair Assessment"  will appear.
You can access all of the tool outputs by clicking on each tab (it can take a little time for your results to be displayed).

## Draft of Data Paper

By clicking on the "Draft of Data Paper" tab, you will have access to the draft of Data Paper presented in an HTML format.
You can either navigate through the Data Paper with the tabs or with the scrollbar on the right and access different elements.

You can at the top of the page download the draft in either an HTML format or an editable docx format.

> ![Download in HTML](./Images/Download_HTML.png""Download in HTML")![Download in docx](./Images/Download_docx.png "Download in docx*)

Reviewer: ```suggestion
![Download in HTML](./Images/Download_HTML.png "Download in HTML")
![Download in docx](./Images/Download_docx.png "Download in docx*)
```
